### PR TITLE
[#1] Initialize project

### PR DESCRIPTION
**Pull request by suho**
## What happened 👀

Setup project from https://github.com/nimblehq/rails-templates
 
## Insight 📝

- Run command: 
```
rails new gooogle-search-ruby -m https://raw..com/nimblehq/rails-templates/master/template.rb
```
 
- Enable addons: 
  - docker
  - nginx
  - bootstrap
  - slim
  - devise

## Proof Of Work 📹

Screen Shot 2021-11-08 at 17 15 37

### CODE REVIEW

File: .phraseapp.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+phraseapp:
```
Comment (hanam1ni) [inline .phraseapp.yml]: I think we don't need to use Phraseapp for IC. This file can be removed

**Review by hanam1ni — APPROVED** [review]

File: .phraseapp.yml (hunk 1 above)
Comment (suho) [reply .phraseapp.yml]: @hanam1ni Fixed in [`ece0c06` (#26)](https://github.com/suho/google-search-ruby/pull/26/commits/ece0c064d824aa98eb9589fd826787aadba2bd60)

File: README.md [hunk 2]
```diff
@@ -1,28 +1,123 @@
...
+docker push $DOCKER_IMAGE:$BRANCH_TAG
+```
+
+- Setup the test database:
+
+```sh
+docker-compose -f docker-compose.test.yml run test bin/bundle exec rake db:test:prepare
+```
+
+#### Semaphore CI 2.0
```
Comment (junan) [inline README.md]: As we will not use Semaphore for CI/CD, I think we can remove it

**Review by junan — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (suho) [reply README.md]: Good catch!

Comment (suho) [reply README.md]: @junan Updated in [`afccc22` (#26)](https://github.com/suho/google-search-ruby/pull/26/commits/afccc223fe6d43e653da5aa94bc7f0abf31298c5)

**Pull request merged**